src/applications/vaos/services/appointment/transformers.js

```diff
@@ -402,7 +404,23 @@ function setContained(appt) {
           ],
         };
 
-        if (appt.sta6aid) {
+        if (tasInfo) {
+          service.location = {
```

Comment:
I think we should go ahead and remove this location property for ATLAS appointments. Setting it to the ATLAS site code means that we try to fetch the site like it's a VA facility, instead of falling back to the `providedBy` property, when displaying the VA facility info in the cancel modal for ATLAS appointments.

From a FHIR standpoint, I also don't think we'd want to include a reference to something that we've included in the `contained` array.

Reply:
Done